api-extractor/src/DocItemLoader.ts

```diff
@@ -229,7 +229,13 @@ export default class DocItemLoader {
       }
     );
 
-    const packageName: string = path.basename(packageJsonFilePath).split('.').shift();
+    // Include the scope name in the cache key
+    // For out test cases, the will be no searching the alias path, 
+    // so we manually include the '@types' scope name
+    const scopeName: string = isTypes ?
+      '@types' : packageJsonFilePath.match(/@\w+/).toString() ;
```

Comment:
I don't understand why you're trying to obtain the package name from the folder path.  That seems unreliable.

Shouldn't you be reading the "name" field from package.json, and/or using parseScopedPackageName()?